C/CPUInfo/bundled/patches/lowercase-windows-include.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/x86/windows/init.c b/src/x86/windows/init.c
```

Comment:
It'd be good to upstream this patch

Reply:
They actually already have fixed that - luckily :-)